docs/advanced/retries.rst

```diff
@@ -0,0 +1,57 @@
+.. _retries:
+
+=======
+Retries
+=======
```

Comment:
What do you think about having subsections that cover the topic of "how to retry":
1. the entire Page Object itself, and
2. Additional Requests

For the 2nd one, we can do that in another PR.

Reply:
I think the first one is covered in this page.

The second one I think we can address in this pull request, but I think it makes more sense in the page about additional requests, and not in a section here.

Reply:
>  but I think it makes more sense in the page about additional requests, and not in a section here.

We could have a dedicated section for it in additional requests and have it linked inside this Retry page (to prevent writing the same things).

I believe it would be much easier for users to find the content if they'd like to perform "retries" for ARs since "Retry" is it's own section, as opposed to clicking "Additional Requests" and stumble in the subsections from the sidebar.

![image](https://user-images.githubusercontent.com/3449761/180138495-f5fb7cec-dbb8-4903-9d6a-99aff2f4cd2b.png)

Reply:
I ended up going for your original approach, just so that I did not had to introduce the scenarios where retries need to be handled in both pages. As a result, I also moved the retry page below the additional requests section in the table of contents.